{"uuid": "d44fb276-83a5-448b-b11d-b55955cf1bf1", "befores": [{"name": "common_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: 'noReset' must be of type boolean\nStacktrace:\nInvalidArgumentError: 'noReset' must be of type boolean\n    at validateCaps (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\capabilities.js:52:11)\n    at parseCaps (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\capabilities.js:165:20)\n    at processCapabilities (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\capabilities.js:211:43)\n    at parseCapsForInnerDriver (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\lib\\utils.js:116:21)\n    at AppiumDriver.createSession (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\lib\\appium.js:325:26)\n    at commandExecutor (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\driver.js:335:9)\n    at AppiumDriver.executeCommand (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\basedriver\\driver.js:349:15)\n    at AppiumDriver.executeCommand (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\lib\\appium.js:549:26)\n    at asyncHandler (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:297:34)\n    at C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:423:15\n    at Layer.handle [as handle_request] (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\YM520\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:335:12)\n", "trace": "  File \"C:\\python3.8\\lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\python3.8\\lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\python3.8\\lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\study\\pythonT\\pyAppium-master\\testcase\\conftest.py\", line 32, in common_driver\n    driver = base_driver.base_driver()\n  File \"C:\\study\\pythonT\\pyAppium-master\\common\\base_driver.py\", line 51, in base_driver\n    driver = webdriver.Remote(f\"http://127.0.0.1:{self.device_info['server_port']}/wd/hub\",\n  File \"C:\\python3.8\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 257, in __init__\n    super().__init__(\n  File \"C:\\python3.8\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\python3.8\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 346, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)\n  File \"C:\\python3.8\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\python3.8\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1699515263599, "stop": 1699515266827}], "start": 1699515263599, "stop": 1699515267112}